IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Pressiometer type_1y3JjznIn4hQPFaA5mCFy3" Name="Pressiometer type" UniqueId="7c0d3b7d-c52c-44ad-a64f-90a17030ff03" PrimaryDataType="IfcLabel" />
```

Comment:
Quite unconventional naming. Pretty sure it is not allowed even (I could be wrong). Propose renaming to "PressiometerType".

Reply:
Fixed in [d57a064](https://github.com/bSI-InfraRoom/IFC-Specification/pull/596/commits/d57a064b92b7b5090dacc930a917ae787e7fb48b)